app/models/miq_provision_virt_workflow.rb

```diff
@@ -1062,10 +1062,23 @@ def ws_vm_fields(values, fields)
     ws_network_fields(values, fields, data)
     ws_customize_fields(values, fields, data)
     ws_schedule_fields(values, fields, data)
+    ws_environment_fields(values, fields, data)
 
     data.each { |k, v| _log.warn "Unprocessed key <#{k}> with value <#{v.inspect}>" }
   end
 
+  def ws_environment_fields(values, _fields, data)
+    # do not parse environment data unless :placement_auto is false
+    if data[:placement_auto].nil? || data[:placement_auto]
```

Comment:
The `:placement_auto` value will be a string here (or nil) when called from the REST API.

This should check: `if data[:placement_auto].to_s != "false"`